Header header
std_msgs/String boxpart
int32 R
int32 G
int32 B